hoodie-client/src/main/java/com/uber/hoodie/index/hbase/HBaseIndex.java

```diff
@@ -98,8 +119,8 @@ private Connection getHBaseConnection() {
   }
 
   /**
-   * Since we are sharing the HbaseConnection across tasks in a JVM, make sure the HbaseConnectio is
-   * closed when JVM exits
+   * Since we are sharing the HbaseConnection across tasks in a JVM, make sure the HbaseConnectio is closed when JVM
```

Comment:
Is this intentional ? Have you configured your intellij correctly ?

Reply:
Imported the code style recommended, but some like these are not covered by it I guess since we there is no strict 1:1 mapping btw that and maven check-style. Sure, will undo this.